public/game/view-models/app-viewmodel.coffee

```diff
@@ -59,7 +60,7 @@ class AppViewModel
 		@occurrenceIndicator.string['<-'] @selector
 
 		@userPanelViewModel = new UserPanelViewModel @user_data
-		@roundTimerViewModel = new TimespanViewModel @countdown, dateTimeFormats['m:ss']
+		@roundTimerViewModel = new CountdownCircleViewModel @countdown, 2 * 60 * 1000, dateTimeFormats['m:ss'], { radius: 40, strokeWidth: 5 }
```

Comment:
We can add another field for puzzle data sent from the service. For now it will always be equal to the round duration constant, that is, 120s. Client apps can then use this data and we change it in the future to be more flexible, front-end code will remain valid. Right?